routes/purchases.js

```diff
@@ -1,6 +1,8 @@
 var purchase = require('express').Router();
 var authHelper = require('../utils/authHelper');
 var Purchase = require('../models/purchaseModel').Purchase;
+var emailHelper = require('../utils/emailHelper');
+var users = require('../models/userModel').User;
```

Comment:
One convention that we've been following is to name the models beginning with a capital letter. This helps us distinguish between the model itself and an instance of the model. In this case, I suggest changing `users` to `User`.